### ISSUE TITLE
EntityDecoder[argonaut.Json] is broken in latest release (0.13.1a)

### ISSUE THREAD

**Issue by jedesah** [http4s/http4s#604]
`e̼` seems to be one of possibly a set of offending characters which trigger a silent error whereas the string literal is no longer the same after having been parsed into json.

I have not investigated yet what changed or might be causing the problem, but I have had to roll my own `EntityDecoder[Json]` in Quasar which simply calls `argonaut.Parse.parse`. That seems to work fine.

The answer might be as simple as a bug in the streaming `jawn` parser.

**Comment by rossabaker** [comment]
Do you have an explicit jawn dependency, or are you on 0.8.4?  This isn't another flavor of https://github.com/non/jawn/issues/45, is it?

**Comment by jedesah** [comment]
We do not not have an explicit dependency on `jawn`, however `dependencyGraph` is telling me that we are on `0.8.3` and somehow `0.8.4` is being evicted.

**Comment by rossabaker** [comment]
Okay. I'd be interested to know if we're causing that, and if an upgrade to 0.8.4 fixes it.  We have a pending bugfix for 0.13.2, but I'll hold off for a bit until we confirm that this isn't also an issue.

**Comment by jedesah** [comment]
Confused by the output of sbt's `evicted` command:

```
[info] 	* org.spire-math:jawn-parser_2.11:0.8.4 -> 0.8.3 (caller: org.http4s:http4s-argonaut_2.11:0.13.1a, org.http4s:jawn-streamz_2.11:0.8.1)
```

I wonder if the same thing will be output if I run the command in `http4s` itself. Could explain why the bug might still not be fixed. Perhaps the reporter of #514 never got around to making sure it actually works now.

**Comment by jedesah** [comment]
Aha! https://github.com/http4s/http4s/blob/master/project/Http4sBuild.scala#L70

That looks like the culprit and perhaps an easy fix!

**Comment by jedesah** [comment]
I'll publishLocal and update on whether it fixes the parsing issue and open a PR (probably either way as we should probably try and keep the evicted list to a minimum).

**Comment by jedesah** [comment]
So that was indeed the fix! PR coming shortly.

**Comment by rossabaker** [comment]
Fix will be in 0.13.2.
